style/light-style.css

```diff
@@ -247,3 +233,67 @@ footer {
   margin-left: 20px;
   margin-bottom: 80px;
 }
+
+@media screen and (min-width: 992px) {
```

Comment:
I have two meadia queries, because have two styles Light style in 1 file, and dark style in another file, they don't repeat, I did that because refactoring code, but, if that is a problem, I can Join the 2 files in a single file, I hope you can respond this comments, because is my last review.